packages/fxa-payments-server/src/routes/Checkout/index.tsx

```diff
@@ -312,155 +312,155 @@ export const Checkout = ({
   return (
     <>
       <Header />
+
       <div className="main-content">
         {newsletterSignupError && <NewsletterErrorAlertBar />}
-        <PaymentErrorView
-          error={subscriptionError}
-          actionFn={() => {
-            setSubscriptionError(undefined);
-            setTransactionInProgress(false);
-          }}
-          className={classNames({
-            hidden: !subscriptionError,
-          })}
-          plan={selectedPlan}
-          showFxaLegalFooterLinks={true}
-        />
+
+        { subscriptionError &&
+          <PaymentErrorView
+            error={subscriptionError}
+            actionFn={() => {
+              setSubscriptionError(undefined);
+              setTransactionInProgress(false);
+            }}
+            plan={selectedPlan}
+            showFxaLegalFooterLinks={true}
+          />
+        }
+
         <PaymentProcessing
           provider="paypal"
           className={classNames({
             hidden: !transactionInProgress || subscriptionError,
```

Comment:
Investigating - I tried to convert this like the others, but the test would fail. 🤨

Reply:
Looked into this. Keep it. Bringing part of our Slack discussion here, more work needs to be done and a follow up ticket to `PaymentProcessing`, which includes other components (e.g. `SubscriptionTitle` and `PaymentLegalBlurb`). It currently fails DRY and needs a few updates made to its tests and what is expected to be/not to be in the document.